packages/experimental/Avatar/FRNAvatar.podspec

```diff
@@ -0,0 +1,22 @@
+require 'json'
+
+package = JSON.parse(File.read(File.join(__dir__, 'package.json')))
+
+Pod::Spec.new do |s|
+  s.name         = 'FRNAvatar'
+  s.version      = package['version']
+  s.summary      = package['description']
+  s.license      = package['license']
+
+  s.authors      = package['author']
+  s.homepage     = package['homepage']
+
+  s.source           = { :git => "https://github.com/microsoft/fluentui-react-native.git", :tag => "#{s.version}" }
+  s.swift_version    = "5"
+
+  s.ios.deployment_target = "11.0"
```

Comment:
Gross it's so old. I just checked RNTester and apparently that's still on iOS 10 though so I guess we can leave for now. Longer term I'd like to adopt a similar approach to Office where we have firm guidelines on what OS versions we support instead of just arbitrarily dropping versions when we find they don't work anymore

Reply:
Yeah =( I kept it as 11 since that's what we have for FluentUI-Apple. That one should be at 13, but I think we have 1 holdover client. Good to bring up with @rdeassis and @harrieshin though! https://github.com/microsoft/fluentui-apple/blob/master/MicrosoftFluentUI.podspec